packages/core/botpress/src/web/views/FlowBuilder/index.jsx

```diff
@@ -15,10 +16,25 @@ import SkillsBuilder from './containers/SkillsBuilder'
 import NodeProps from './containers/NodeProps'
 
 import { switchFlow } from '~/actions'
+import { getDirtyFlows } from '~/reducers'
 
 const style = require('./style.scss')
 
 class FlowBuilder extends Component {
+  componentWillUnmount() {
+    const { pathname } = this.props.history.location
+    const hasDirtyFlows = !_.isEmpty(this.props.dirtyFlows)
+    const hasUnsavedChanges = !/flows/g.exec(pathname) && !window.BOTPRESS_FLOW_EDITOR_DISABLED && hasDirtyFlows
```

Comment:
This won't work on some urls, e.g.

`/modules/custom-flows`

Reply:
@slvnperron  @emirotin I think it's normal flow if not I need more clear explanation for this issue https://github.com/botpress/botpress/issues/516

Reply:
@slvnperron OK, it seems there are nested flows which I didn't notice before. Will fix later

Reply:
Hi @slvnperron , I tried reproduce behaviour you described. What I did:
a.
1) I created skill
2) Went to `See flow`
3) Did some changes
4) Clicked some link
A confirmation message occurred - which is fine.

b.
1) I created skill
2) Went to `See flow`
3) Didn't do any changes and clicked some link
A confirmation message didn't occur - which is also fine.

You mentioned navigating to `/modules/custom-flows` shouldn't trigger confirmation window. But how can I navigate to URL like this? Is it some different package?

Reply:
`/modules` URLs are from installable packages indeed. @slvnperron can you please elaborate a bit about `custom-flows`. Is it some specific package?

Reply:
Oh wait I understood. It's npt about other URLs, it's actually about the regexp you've used to test the URL. It should be something like `/^\/flows\//`